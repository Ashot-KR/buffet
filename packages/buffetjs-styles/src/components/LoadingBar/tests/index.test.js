import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import LoadingBar from '../index';

const renderComponent = () => shallow(<LoadingBar />);

describe('<LoadingBar />', () => {
  it('should not crash', () => {
    renderComponent();
  });

  it('should match snapshot', () => {
    const tree = renderer.create(<LoadingBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
