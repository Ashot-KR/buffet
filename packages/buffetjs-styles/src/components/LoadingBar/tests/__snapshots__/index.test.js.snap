// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingBar /> should match snapshot 1`] = `
.c0 {
  height: 6px;
  width: 20%;
  margin-top: 10px;
  position: relative;
  overflow: hidden;
  background-color: #F3F3F4;
  border-radius: 2px;
}

.c0:before {
  display: block;
  position: absolute;
  content: '';
  left: -200px;
  width: 200px;
  height: 6px;
  background-color: rgb(227,227,231);
  -webkit-animation: dBQFcQ 2s linear infinite;
  animation: dBQFcQ 2s linear infinite;
}

<div
  className="c0"
/>
`;
